source/release-notes/3-1-4.rst

```diff
@@ -0,0 +1,42 @@
+.. index:: Crafter CMS 3.1.4 Release Notes
+
+-----------------
+Crafter CMS 3.1.4
+-----------------
+
+^^^^^^^^^^^^
+New Features
+^^^^^^^^^^^^
+
+* Added support for Crafter CMS plugins to be listed in `Crafter Marketplace <https://github.com/marketplace/crafter-marketplace>`_ simply by tagging your project on GitHub.  Follow the instructions :ref:`here <marketplace_create_plugins>` to create your own plugins and list it in the `Crafter Marketplace <https://github.com/marketplace/crafter-marketplace>`_
+* Added support for flattened content XML through a new data source ``Embedded Content`` to be used for creating inline components
+* Added a date-time picker inside the targeting dialog in the Website Editorial blueprint
+* Added ``Publishing Queue`` tab to ``Site Config`` ➜ ``Publishing``
+* Added :ref:`code-editor-configuration` for adding/editing snippets
+* Added a logrotate sidecar image for docker/kubernetes
+
+^^^^^^^
+Updates
+^^^^^^^
+
+* Upgraded Jackson library to 2.9.10 for various Crafter modules
+* Upgraded Quartz library to 2.3.2 for various Crafter modules
+* Added support for passing parameters to blueprints during site creation
+* Added support for cache warming and cache double buffering
```

Comment:
Added support for automatic Engine cache warming and double-buffering in Serverless mode